pkg/config/ovn/ovn.go

```diff
@@ -196,3 +197,73 @@ func IsOVNKubernetesInternalInterface(name string) bool {
 
 	return excludedInterfacesRegexp.MatchString(name)
 }
+
+func HasDefaultGateway(family int) (bool, error) {
```

Comment:
The logic here looks OK. I think the logging needs some work.

We expect to need to operate in environments with no gateway, with multiple NICs, etc. Therefore, I'm not sure it makes sense to warn as we search for the default gateway.

Can we log at each step using info level messages like "Did not find gateway for route..." and then when the gateway is found say something like "Found gateway for route..." so that someone debugging a system can see how MicroShift is interpreting the system configuration?

Reply:
Added logs.